pages/projects/index.vue

```diff
@@ -41,14 +38,24 @@
 <script>
 import { createNamespacedHelpers } from 'vuex'
 
-const { mapGetters, mapActions, mapMutations } = createNamespacedHelpers(
-  'projects'
-)
+const { mapGetters, mapActions } = createNamespacedHelpers('projects')
 
 export default {
   data: () => {
     return {
       dialog: false,
+      newProject: {
```

Comment:
supposed to be in the `project` store, isn't?

Reply:
The logic here looks like this: when user clicks on New Project button, the new project is created and is added to the `projects` array in the db.json and in the application state. As the `projects` array is being modified, I placed the functions in the store/projects.